src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -517,55 +523,56 @@ private AIState structureStep()
 
     private AIState decorationStep()
     {
-        if(!goToConstructionSite())
+        if (!goToConstructionSite())
         {
             return this.getState();
         }
-
-        if(job.getSchematic().doesSchematicBlockEqualWorldBlock() || job.getSchematic().getBlock().getMaterial().isSolid() /*|| job.getSchematic().getBlock() == Blocks.air*/)
+        //|| job.getSchematic().getBlock() == Blocks.air
+        if (job.getSchematic().doesSchematicBlockEqualWorldBlock()
+            || job.getSchematic().getBlock().getMaterial().isSolid())
         {
             //findNextBlock count was reached and we can ignore this block
             return findNextBlockNonSolid();
         }
 
         worker.faceBlock(job.getSchematic().getBlockPosition());
-        Block block = job.getSchematic().getBlock();
+        Block       block    = job.getSchematic().getBlock();
         IBlockState metadata = job.getSchematic().getBlockState();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'metadata' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)